features/step_definitions/main_landing_page_steps.rb

```diff
@@ -0,0 +1,8 @@
+When(/^I visit the main landing page$/) do
+  visit root_path
+end
+
+When(/^I click the "([^"]*)" link$/) do |link|
+  click_link link
+  %q{ I visit the main landing page }
```

Comment:
No no no. If you click the link and it takes you to the main landing page, then you should be on the main landing page. You're clicking a link and then redirecting in the same step.